mscore/file.cpp

```diff
@@ -2540,76 +2540,80 @@ bool MuseScore::saveSvg(Score* score, const QString& saveName)
             p.translate(-r.topLeft());
 
       foreach (Page* page, score->pages()) {
```

Comment:
`foreach(a, b)` is the Qt way, `for (a : b)` is the C++11 way and, as per https://musescore.org/en/developers-handbook/musescore-coding-rules#for, to be peferred.